### PR TITLE
Feature/traffic sign

### PR DESCRIPTION
**Pull request by hellotechx**
# Issue

In OSM data "traffic signals" is an attributes on node, but in Unidb data "traffic signals" is an attributes on relations.

When process node, need to parse relations to get info of traffic signals. 

Modify scripts to support retrieve traffic signals info from relations when process node.

### CODE REVIEW

File: profiles/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Changelog
```
Comment (wangyoucao577) [inline profiles/CHANGELOG.md]: Excellent format. We can follow the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) for our osrm-backend release too.

File: profiles/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [Unreleased]
+
+## 2020-03-17
```
Comment (wangyoucao577) [inline profiles/CHANGELOG.md]: After read [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), I think the date should only applied on versions, not every action.

Comment (hellotechx) [reply profiles/CHANGELOG.md]: > After read [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), I think the date should only applied on versions, not every action.

You are right. The original template does have version https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md
Do you suggest remove all date from Change logs, for current, only keep 'Added', 'Changed' and 'Remove' list instead?

Comment (wangyoucao577) [reply profiles/CHANGELOG.md]: After check https://github.com/Project-OSRM/osrm-backend/blob/master/CHANGELOG.md, I think the best practice maybe write down many contents when initialize, then add oneline to describe the change for every PR, also the PR/ISSUE ID better to be record.

Comment (hellotechx) [reply profiles/CHANGELOG.md]: After discussion, remove "added", "removed" and "deleted". 
Link PR to each change.

**Pull request merged**